doc/instruction_decode_execute.rst

```diff
@@ -165,3 +162,13 @@ Source File: :file:`rtl/ibex_load_store_unit.sv`
 
 The Load-Store Unit (LSU) interfaces with main memory to perform load and store operations.
 See :ref:`load-store-unit` for more details.
+
+.. rubric:: Footnotes
+
+.. [#B_draft] Ibex fully implements draft version 0.92 of the RISC-V Bit Manipulation Extension.
+   This extension may change before being accepted as a standard by the RISC-V Foundation.
```

Comment:
accepted => ratified, as that seems to be the preferred terminology in these discussions

Reply:
Changed, thanks Alex.